addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -142,7 +142,7 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
                 this.update();
             }).bind(this));
 
-            var url = session.debug ? $.param.querystring(tour.url, {debug: session.debug}) : tour.url;
+            var url = tour.url;
             window.location.href = window.location.origin + url;
```

Comment:
Lol :p Someone pays you to keep lines ? :p

Reply:
Yeah, looking at the diff this looks stupid, did not pay attention while removing the ternary condition